docs/source/gp.rst

```diff
@@ -244,7 +244,7 @@ also need to include the additional arguments, :code:`X`, :code:`y`, and
 This second block produces the conditional distributions.  Notice that extra 
 arguments are required for conditionals of :math:`f1` and :math:`f2`, but not
 :math:`f`.  This is because those arguments are cached when calling 
-:code:`.marginal_likelihood` was called on :code:`gp`.
+:code:`.marginal_likelihood` on :code:`gp`.
```

Comment:
I think the original is correct here: "...are cached when calling marginal likelihood on gp."

Reply:
My thinking when I read the sentence was that the action "to be called" was repeated and that one copy should be removed, but since I'm not familiar with the code at all it's completely possible I misinterpreted the statement, and "calling `.marginal_likelihood`" (as a gerund) is what is actually called on `gp`.

Reply:
I see. Could say “are cached when marginal_likelihood is called on gp.” instead.